build.xml

```diff
@@ -70,10 +66,8 @@
 	<property name='jsapi-inventory.file' value='${build.docs.dir}/jsapi-inventory.zip'/>
 	<property name='jsapi-changelog.file' value='${build.docs.dir}/jsapi-changelog.zip'/>
 
-    <property name='build.img.dir' value='${build.web.dir}/img'/>
-	<property name="build.css.file" value="images"/>
-	<property name="build.css.ext" value=".css"/>
-    <property name="build.css.filename" value="${build.css.file}${build.css.ext}"/>
+    <property name="build.css.ext" value=".css"/>
+    <!--property name="build.css.filename" value="${build.css.file}${build.css.ext}"/ -->
```

Comment:
we have commented some code in below part also which requires this variable so I have commented this also

we can remove in future